tests/scripts/minikube.sh

```diff
@@ -81,6 +81,8 @@ case "${1:-}" in
     minikube start --memory=3000 --kubernetes-version ${KUBE_VERSION} --extra-config=apiserver.Authorization.Mode=RBAC
     wait_for_ssh
     enable_roles_for_RBAC
+    minikube ssh "sudo mkdir /mnt/sda1/var/lib/rook;sudo ln -s /mnt/sda1/var/lib/rook /var/lib/rook"
```

Comment:
why is this needed?

Reply:
The default path of `/var/lib/rook` will only work for our minikube environment if we create this link